setup.cfg

```diff
@@ -0,0 +1,14 @@
+[flake8]
+# Match black line length
+max-line-length = 88
+# E501: Line too long
+# W503: Line break occurred before a binary operator
+# E203: See https://github.com/PyCQA/pycodestyle/issues/373
+ignore = E501, W503, E203
```

Comment:
```suggestion
max-line-length = 138
# W503: Line break occurred before a binary operator
# E203: See https://github.com/PyCQA/pycodestyle/issues/373
ignore =W503, E203
```
Set the `max-line-length` instead of ignoring line length altogether with E501.